organization/_posts/2019/2019-05-29-packaging.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "HSF Packaging Meeting #34, 29 May, 2019"
+layout: meetings
+---
+
+# {{page.title}}
+
+## Agenda
+[<span class="underline">https://indico.cern.ch/event/819329/</span>](https://indico.cern.ch/event/819329/)
+
+Participants: Ben Morgan, Patrick Gartung, Graeme Stewart, Javier Cervantes, Chris Burr,
+Pere Mato, Dimitri, Geri Ganis, Guilherme Amadio, Marco Clemencic, Chris Green, Elizabeth Sexton-Kennedy
+
+## Introduction (Graeme)
+- Ongoing Projects: FCC/LCG
+  - LCG stack with Spack (+ CERN summer student from June/July)
+    - Javier: some issues with ROOT, but everything below o.k.
+  - CHEP abstract submitted (+ wider WG topics), additional collaboration welcome
+- Any other CHEP abstracts submitted?
+  - Chris Burr: one on Conda/ROOT + CVMFS
+  - Guilherme: Maybe one from Gentoo/Portage
+  - Patrick/Chris: One on SpackDev
+- Next meeting options: 19th/26th June/3 July
+  - 19th maybe too soon?
+  - 26th awkward due to many people travelling
+  - 3rd July looking best
+
+## Spack/SpackDev (Chris Green)
+- Improvements to Spack core
+  - "VISUAL" env var for recipe editing
+  - Better saving of environment plus PATH-like var manipulation
+  - Better CMake builds
+- Core recipe improvements
+  - Inc: mysql, mariadb, postgresql, intel-tbb, cppunit, gl2ps, range-v3, nix, boost, sqlite, numpy
+  - Reconciled FNAL root recipe with upstream
+  - Resolved "-isystem" issue with ROOT/Spack
+- Other news
+  - Work Items doc updated to version 0.4
+  - Bi-weekly production meetings with Peter Scheibel (Spack principal), inc. discussion on items in above doc, with WIP/AIP on requirements
+  -  AIP to merge SpackDev into core, telcon with Elizabeth Fischer (Spack contributor) who’s working on similar topic
+- SpackDev progress
+  - Accuracy/Speed improvements for consistent package checkout
+  - Package names can have hyphens
+  - Accommodate changes to upstream Spack
+  - Print "spec tree" for packages specified for development
+  - Input DAG (everything need for a release) can have several top-level packages with co-concretization
+  - Package checkout improvements, e.g. require that development packages come from VCS systems.
+  - Global build can handle tests in parallel
+- MVP1a
+  - Integration testing round that identified previously mentioned updates
+  - LArSoft now building with Art 3.02.04 (but will go to 3.02.05 for MVP)
+  - Remaining issue with "wirecell" package, need to support Git submodules
+    - Spack supports submodules, but wirecell needs a additional manual step.
+    - Guilherme: Could create tarballs? Chris: Would imply FNAL becomes maintainer of those tarballs. Fix locally in Spack to get MVP progressing
+- Chris: some issues still require updates to the core concretization system, aim is end of US fiscal year, but may slip.
```

Comment:
I think "US fiscal year" is too obscure for non-US people. I suggest "end of September".